cpp/models/abm/world.cpp

```diff
@@ -111,11 +114,11 @@ void World::migration(TimePoint t, TimeSpan dt)
             if (nonempty) {
                 auto target_type = rule.first(*person, t, dt, m_migration_parameters);
                 Location* target = find_location(target_type, *person);
-                if (target != &get_location(*person)) {
-                    if (target->get_testing_scheme().run_scheme(*person, m_testing_parameters)) {
+                if (run_testing_schemes(*person, *target)) {
```

Comment:
singular!

Reply:
Resolved through the implementation of testingStrategy.